Bindings/Java/tests/TestEditProperties.java

```diff
@@ -7,6 +7,7 @@
   public static void main(String[] args) {
     try {
         Model model = new Model("gait10dof18musc_subject01.osim");
+        model.finalizeFromProperties();
```

Comment:
This reflects the GUI workflow where initSystem is called right after construction, so I'd prefer to remove the call to finalizeFromProperties here to make sure the call sequence with only initSystem continues to work.